pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/DDPRequestRouteTest.java

```diff
@@ -136,7 +136,7 @@ public void checkForParticipant() throws Exception {
                                 "https://track.easypost.com/djE6dHJrXzY4NGJmYzU3ZjM5OTQ1Zjg5MjEzOGRmMWVmMjI1NWZl", null, 12L, false, "",
                                 12L, null, 12L, "so what", "mf_testLabel", false, "shp_f470591c3fb441a68dbb9b76ecf3bb3d", 12L, null,
                                 "44445", false, "NOT FOUND", null, null, null, null, 0L, false, null, null, null, null, null, null, null,
-                                null, null, null));
+                                null, null, null, null));
```

Comment:
The only use of this new `displayName` in a test appears to only check cases where the value is null.  I can't tell if we have tests that hit the new display name with a non-null value.